packages/fuelet-wallet/jest.setup.ts

```diff
@@ -0,0 +1,35 @@
+import { performance } from 'node:perf_hooks';
```

Comment:
Can we have just a single infra for all this tests? Looks like the code is been repeated eveytime, maybe we can add vitest istead of jest to help on having a single test config for all the pacakges.

Reply:
i'll check this out

Reply:
https://github.com/FuelLabs/fuel-connectors/pull/1/commits/a6832b894299b492d1000548a86cc00849aa8380